### PR TITLE
micmute: Add version 0.7.9

### PR DESCRIPTION
**Pull request by SaifAqqad**

### CODE REVIEW

File: bucket/micmute.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "config.json"
+    ],
+    "uninstaller": {
+        "script": [
+            "$FileName = \"$env:APPDATA\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\MicMute.lnk\"",
+            "if (Test-Path $FileName) {",
+            "   Remove-Item $FileName",
+            "}"
+        ]
+    },
```
Comment (Ash258) [inline bucket/micmute.json:30]: Why??

File: bucket/micmute.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    "shortcuts": [
+        [
+            "MicMute.exe",
+            "MicMute"
+        ]
+    ]
```
Comment (Ash258) [inline bucket/micmute.json:36]: "homepage": "https://github.com/SaifAqqad/AHK_MicMute",
    "description": "Microphone controller using keyboard/mouse shortcuts",
    "homepage": "https://github.com/SaifAqqad/AHK_MicMute",
    "license": "Unlicense",
    "url": "https://github.com/SaifAqqad/AHK_MicMute/releases/download/0.7.9/MicMute.exe",
    "hash": "5f0e6b5717b0697c50e8a9a6d68e830ffa6aeff331211211eacec9249f2b1726",
    "pre_install": "if (!(Test-Path \"$persist_dir\\config.json\")) { New-Item \"$dir\\config.json\" | Out-Null }",
    "uninstaller": {
        "script": "Remove-Item \"$env:APPDATA\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\MicMute.lnk\" -ErrorAction SilentlyContinue"
    },
    "shortcuts": [
        [
            "MicMute.exe",
            "MicMute"
        ]
    ],
    "persist": "config.json",
    "checkver": "github",
    "autoupdate": {
        "url": "https://github.com/SaifAqqad/AHK_MicMute/releases/download/$version/MicMute.exe",
        "hash": {
            "url": "https://github.com/SaifAqqad/AHK_MicMute/releases/tag/$version",
            "regex": ">$sha256</code"
        }
    }
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/micmute.json (hunk 1 above)
Comment (SaifAqqad) [reply bucket/micmute.json:30]: To make sure everything is removed when uninstalling it.

Comment (SaifAqqad) [reply bucket/micmute.json:30]: The whole startup thing will be handled differently in the next version (with task scheduler), but for now this is kind of necessary

**Pull request closed without merging**